src/applications/personalization/profile360/selectors.js

```diff
@@ -24,3 +24,17 @@ export const directDepositAddressIsSetUp = state => {
     addressInfo?.stateCode
   );
 };
+
+export const directDepositIsBlocked = state => {
+  const controlInfo =
+    directDepositInformation(state)?.responses?.[0]?.controlInformation || {};
+  // If all of these flags are not strictly equal to `true` we want to block
+  // access to the direct deposit feature. ie, we want to err on the side of
+  // blocking people from accessing direct deposit rather than err on the side
+  // of letting people have access to the feature.
+  return (
+    controlInfo.isCompetentIndicator !== true ||
+    controlInfo.noFiduciaryAssignedIndicator !== true ||
+    controlInfo.notDeceasedIndicator !== true
+  );
+};
```

Comment:
This LGTM - only nit is we could remove all of the double negatives:

```
const directDepositAvailable = state => {
  return (
    control.Info.isCompetentIndicator ||
    controlInfo.noFiduciaryAssignedIndicator ||
    controlInfo.notDeceasedIndicator
  )
}
```

Reply:
So I didn't want to do that because I wanted to check that flags like `isCompetentIndicator` is explicitly `true` and not just truthy. Probably overkill, but better to be safe.